exercises/practice/rotational-cipher/.meta/template-data.yaml

```diff
@@ -0,0 +1,33 @@
+unit: module
+
+properties:
+  rotate:
+    test: |-
+      sprintf(q:to/END/, %case<input><text>.Str, %case<input><shiftKey>.Int, %case<expected>.Str, %case<description>.raku);
+      subtest "%1$s" => {
+        cmp-ok(
+            caesar-cipher("%1$s", %2$s),
+            "eq",
+            "%3$s",
+            %4$s,
+        );
+        if %2$s == 13 {
```

Comment:
I've made some changes to this to avoid having dead code. I'll push them up outside of my working hours.